packages/shared/src/graphql/feedSettings.ts

```diff
@@ -78,14 +78,14 @@ export const SEARCH_TAGS_QUERY = gql`
 `;
 
 export const FEED_SETTINGS_QUERY = gql`
-  query FeedPreferences {
+  query FeedPreferences($feedId: ID) {
```

Comment:
`feedId` is optional in this and all new queries. If not provided by default API loads it from `ctx.userId` as before.